algopy_testing/examples/auction/test_contract.py

```diff
@@ -0,0 +1,146 @@
+from collections.abc import Generator
+from typing import Any
+from unittest.mock import MagicMock
+
+import pytest
+from algopy import Account, Asset, UInt64, gtxn
+from algopy_testing import TestContext, blockchain_context
+
+from .contract import AuctionContract
+
+
+@pytest.fixture()
+def context() -> Generator[TestContext[Any], None, None]:
+    with blockchain_context() as ctx:
+        yield ctx
+
+
+@pytest.fixture()
+def contract() -> AuctionContract:
+    return AuctionContract()
+
+
+def test_opt_into_asset(
+    context: TestContext[Any], contract: AuctionContract, mocker: MagicMock
+) -> None:
+    # Arrange
+    dummy_address = "A" * 58
+    context.global_state.creator_address = Account(dummy_address)
```

Comment:
I think `global_state` is confusing and could be confused with global strorage. Let's call it `global`?

Reply:
`global` is a keyword in python so wouldn't be able to use that `global_` would be valid